### PR TITLE
Datesandquantity

### PR DESCRIPTION
**Pull request by chriztso**
Changes made
-Added shipping estimation descriptions with live dates and live countdown. 
-Also added In Stock text
screen shot 2019-02-13 at 9 52 08 am
screen shot 2019-02-13 at 10 21 13 am

### CODE REVIEW

File: client/src/__snapshots__/Dates.spec.js.snap [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+<div
+  className="expectedDates"
+>
+  <div
+    className="free"
+  >
+    <span
+      className="bold"
+    >
+      Get it
```
Comment (andrewgeorgemitchell) [inline client/src/__snapshots__/Dates.spec.js.snap:13]: Why is this spread out on like 30 lines?

Comment (chriztso) [reply client/src/__snapshots__/Dates.spec.js.snap:13]: Not exactly sure, should I edit the format?

File: client/src/Dates.jsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  const actualWeekday = daysOfWeek[weekday];
+  const fourDaysLater = daysOfWeek[weekday + 4];
+  const hour = date.getHours();
+  const minutes = date.getMinutes();
+  return (
+    <div className="expectedDates">
+      <div className="free">
+        <span className="bold">
+          Get it
+          {' '}
```
Comment (andrewgeorgemitchell) [inline client/src/Dates.jsx]: Consider merging this entire span into one or two long {`I am ${something} that is all on one line`} I know airbnbs style makes it feel like you can't put stuff on one line but you can it just has to be either a line of entirely stuff between {} or just a line of strings

**Review by andrewgeorgemitchell — APPROVED** [review]
Since you are trying to essentially make your own date handling functions it may be worth it to export all your date logic out to a date.js file that lives inside a shared folder that you import into any react component that has to deal with dates, I know you kind of already do this by using a Dates React Comp, but the dates.jsx file is pretty big for what it does, so maybe just creating that service and then just allowing helper function to be imported from that service would look cleaner. 

Also look through my comments throughout because you may want to refactor your text inside your html to be more readable, because right now it is really unreadable because you have it split on so many lines

File: client/src/Dates.jsx (hunk 2 above)
Comment (chriztso) [reply client/src/Dates.jsx]: Sure, sounds good, I'll go ahead and refactor to be on less lines.

**Pull request merged**